### ISSUE TITLE
Supplies disappeared

### ISSUE THREAD

**Issue by amunoz20** [Wyqer/kp_liberation#130]
### Basic Information
**Mission version:**
0.952
**Map used:**
Tanoa
**Mods used:**
RHS, CUP, FIR Planes, USAF, Advanced AI command, rappeling, towing, and other skins mods
**Dedicated or local game?**
Local
**GitHub or Workshop mission file?**
Workshop file

### Individual Things
**Did you edit anything within the mission files?**
Yes

**If yes, what did you edit exactly?**
I added vehicles to apex preset file

### Short Issue Description
Supplies never come back ater you cancel a vehicle build from the build menu.

### Expected Behaviour
If you choose a vehicle and then cancel the build, the resources should refund.

### Steps to Reproduce the Issue
Open Build Menu
Select light vehicles
Choose one of them (I selected a vehicle in the original preset and not added by me)
Rotate it, move it
Cancel the build

**Comment by Wyqer** [comment]
Just tested it after your report and I don't experience any issues with that. Works like intended.
You hit build and get the preview (the resources are payed already). If you hit cancel, the resources are back in the storage area. No problems there.
What things did you edited exactly?
If you try it with an unedited missionfile you should also not experience this behaviour.

**Comment by amunoz20** [comment]
Okay then probablly is something on my end. I only added a couple of vehicles to the air_vehicles array. So it shouldnt break anything else. Last night when I was playing I hit build and got the preview (resources payed) but when I hit cancel the resources never came back.

Later today I will try to reproduce this problem again on same mission edited and unedited to check for the same behaviour.

I also noticed that the HEMT vehicle couldnt tow one of the technicals (civilian trucks mounted with a .50 cal). I will double check it today.
